packages/react-instantsearch-core/src/index.ts

```diff
@@ -8,6 +8,10 @@ export { default as translatable } from './core/translatable';
 
 // Widgets
 export { default as Configure } from './widgets/Configure';
+export {
+  // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
can we change the rule, like in InstantSearch.js?

Reply:
Will do.